packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -177,13 +178,33 @@ export class DevWorkspaceClient extends WorkspaceClient {
     const name = createdWorkspace.metadata.name;
     const workspaceId = getId(createdWorkspace);
 
+    // do we have inline editor as part of the devfile ?
+    const inlineEditorYaml = ((devfile as any).attributes && (devfile as any).attributes['che-editor.yaml']) ? safeLoad((devfile as any).attributes['che-editor.yaml']) as devfileApi.Devfile : undefined;
+    
     const devfileGroupVersion = `${devWorkspaceApiGroup}/${devworkspaceVersion}`;
     const devWorkspaceTemplates: devfileApi.DevWorkspaceTemplateLike[] = [];
-    for (const pluginDevfile of pluginsDevfile) {
+
+    // do we have a custom editor specified in the repository ?
+    const cheEditorYaml = optionalFilesContent['.che/che-editor.yaml'] ? safeLoad(optionalFilesContent['.che/che-editor.yaml']) as devfileApi.Devfile : undefined;
```

Comment:
well, I think that ide,plugins resolving logic deserves its own file, but it's not your duty, we'll handle that.